spec/index.bs

```diff
@@ -6396,8 +6402,198 @@ enum GPUVertexFormat {
 };
 </script>
 
```

Comment:
Here I think there should be an explanation of what "WGSL type" means - i.e. what exactly you meant by "ideal" format in the PR summary.

Reply:
Done